src/app/dashboard/dashboard.component.scss

```diff
@@ -1,9 +1,10 @@
 @import 'global_variables';
 @import 'global_states';
 
-.dash-wrapper {
-  padding: 0.925rem 4.5% 5rem;
 
+:host {
+  padding: 0.925rem 4.25% 5rem;
+  min-width: 91.5%;
```

Comment:
Nice to see widths and percentages making their way more and more into our styles!